src/test/java/com/ays/assignment/service/impl/AssignmentServiceImplTest.java

```diff
@@ -93,4 +103,91 @@ void givenAssignmentId_whenAssignmentNotFound_thenThrowAysAssignmentNotExistById
         Mockito.verify(identity, Mockito.times(1))
                 .getInstitutionId();
     }
+
+    @Test
+    void givenAssignmentListRequest_whenAssignmentStatusIsAvailable_thenReturnAysPageAssignmentResponse() {
+
+        // Given
+        String mockInstitutionId = AysRandomUtil.generateUUID();
+
+        AssignmentListRequest assignmentListRequest = new AssignmentListRequestBuilder().withValidValuesForStatus().build();
+
+        List<AssignmentEntity> mockAssignmentEntities = Collections.singletonList(new AssignmentEntityBuilder()
+                .withValidFields().build());
```

Comment:
Burada `Collections.singletonList()` yerine `List.of()` kullanmamız daha efektif olabilir

Reply:
Değiştirdim. @agitrubard 
Commit: `d28950bc14bfef1bdc7dd71cbafc63a1e0d445e6`